docs/articles/rtl.md

```diff
@@ -0,0 +1,53 @@
+All Nebular components were updated to support RTL out of the box.
+
+Now every component, that previously could be positioned via some setting, besides old `left` and `right` options, also support logical `start` and `end` properties, similar to flexbox. Value of `start` and `end` depends on current layout direction. For LTR it's left, for RTL - right.
+
+You can set document direction via last parameter `forRoot` method of `NbThemeModule`. Supported values are 'ltr' and 'rtl'.
+```typescript
+@NgModule({
+  imports: [
+    NbThemeModule.forRoot(nbThemeOptions, nbJSThemes, nbMediaBreakpoints, 'rtl')
+  ]
+})
+```
+Default value is `ltr`.
+<div class="note note-info">
+  <div class="note-title">Note</div>
+  <div class="note-body">
+    1. Works only in layout component
```

Comment:
Components must be placed inside of the `<nb-layoyt></nb-layout>` component to correctly support language direction.